src/components/WizardModal/WizardModal.test.jsx

```diff
@@ -156,32 +156,36 @@ describe('WizardModal', () => {
     expect(wrapper.find(Loading)).toHaveLength(1);
   });
   it('clicking on progressIndicator steps will render related content', () => {
-    render(<WizardModal {...commonWizardProps} steps={[
```

Comment:
Was this just a reformat? I *think* it looks identical but its hard to tell

Reply:
yep 👍